app/lib/features/settings/pages/labs_page.dart

```diff
@@ -66,16 +66,6 @@ class SettingsLabsPage extends ConsumerWidget {
             SettingsSection(
               title: Text(L10n.of(context).apps),
               tiles: [
-                SettingsTile.switchTile(
-                  key: tasksLabSwitch,
```

Comment:
hmpf.... I am not really happy about this approach of just removing the labs flag. IMHO this isn't even in the state to even have the labs-flag default-on yet, we are still missing several basics and there are still going to be major changes ... the point of the labs is to make clear that exactly that is the case to the user: default off for unstable features, default-on for features we consider complete and are just doing minor bug fixes on, removing the labs once we consider this stable. This is anything but stable yet.

Reply:
Yupp, agree with you. Let me roll-back it to lab feature again.